src/app/view-modes/text-versions/text-versions.component.ts

```diff
@@ -27,6 +28,15 @@ export class TextVersionsComponent implements OnInit {
     map(({ editionLevels }) => editionLevels[0]),
   );
 
+  public get versionBtnLabel(): { label: string, additionalClasses: string, title: string, icon?: EvtIconInfo } {
```

Comment:
Sorry, bad suggestion the previous time. "Label" can be omitted since the return type is clearly not a label